app/controllers/pet_applications_controller.rb

```diff
@@ -0,0 +1,31 @@
+class PetApplicationsController < ApplicationController
+
+  def create
+    applicant = Application.find(params[:id])
+    pet_application = PetApplication.create!({
```

Comment:
You don't need to store this (line 5) in a variable because you are just creating a new association but never call `pet_application` anywhere else.